core/modules/dblog/dblog.admin.inc

```diff
@@ -339,10 +339,9 @@ function dblog_filter_form($form) {
   $filters = dblog_filters();
 
   $form['filters'] = array(
-    '#type' => 'fieldset',
-    '#title' => t('Filter log messages'),
-    '#collapsible' => TRUE,
-    '#collapsed' => empty($_SESSION['dblog_overview_filter']),
+    '#type' => 'details',
```

Comment:
I believe that these have been added for the purposes of testing. We have not agreed to start replacing existing fieldsets in core with `<details>`, so these should be removed before merging.